<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="radioButtonC">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Compress</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonD">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Decompress</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowse">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Password:</string>
    </property>
    <widget class="QLineEdit" name="pwdtxt1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwdtxt2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QDialogButtonBox" name="buttonOkC">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
